Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -17,6 +17,12 @@ def teardown
     @formula_files.each(&:unlink)
   end
 
+  def skip_unless_test_cmd_taps
```

Comment:
We have a very similar method in `test/testing_env.rb` named `needs_compat`. Would you consider renaming this one to `needs_test_cmd_taps` for consistency? (I don't disagree that the currently used method name is appropriate if viewed in isolation.)